recognition/advanced_tree_search.py

```diff
@@ -817,7 +831,7 @@ class BuildGlobalCacheJob(rasr.RasrCommand, Job):
     Standalone job to create the global-cache for advanced-tree-search
     """
 
-    def __init__(self, crp, extra_config=None, extra_post_config=None):
+    def __init__(self, crp, extra_config=None, extra_post_config=None, mem=12):
```

Comment:
are you sure this much memory is needed? here some small amount should actually be suficcient as we do not need to load the LMs into memory.

Or could it be that in the crp that are given to this job still some LM is present, which is loaded into memory at runtime for this job. In that case you could test setting the `language_model_config.type = "zerogram"` to completely ignore it. (Not 100% sure this works in any case, please test)

Reply:
> It seems to me like it should be the job's duty to use a dummy LM then, in case RASR wants one but doesn't actually use it.

yes, I think as long as the Lm used has the same vocab then it should be fine to replace the LM in this Job with a dummy.

Reply:
What about this. @michelwi any more comments? Otherwise I think we can skip this for now.

Reply:
yes, we can keep it this way.